src/components/PeopleItem/PeopleItem.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {TouchableOpacity, Text, View} from 'react-native';
+import getGlobalStyles from '~/helpers/GlobalStyles';
+import Colors from '../../helpers/Colors';
+import {getInitials} from '../../helpers/Functions';
+import Image from '../Image';
+import Icon from 'react-native-vector-icons/Fontisto';
+import getStyles from './PeopleItem.styles';
+
+const PeopleItem = ({avatar, firstName, lastName, gender, quote, address}) => {
+  const styles = getStyles();
+  const globalStyles = getGlobalStyles();
+
+  const renderAvatar = () => {
+    if (avatar) {
+      return <Image source={avatar} size={125} borderRadius={30} />;
+    } else {
+      return (
+        <View style={styles.avatarInitialsContainer}>
+          <Text style={styles.avatarInitials}>
+            {getInitials(firstName + ' ' + lastName)}
+          </Text>
+        </View>
+      );
+    }
+  };
+
+  return (
+    <TouchableOpacity
+      onPress={() => {
+        console.log('PeopleItem component pressed!');
```

Comment:
Don't handle the action in the component itself. Components should not have logic. This callback should be passed to the parent screen and the action will be handled in the container.